src/components/list/ProjectsList.tsx

```
@@ -14,11 +14,35 @@ export default function ProjectsList({ projects, isLoading }: {
         daysLeft: number
         views: number
-    }[]
+    }[] | undefined
     isLoading: boolean
 }) {
     const renderList = () => {
-        if (projects.length === 0 && isLoading) return <Spinner message="프로젝트를 불러오는 중입니다." />
+        if (projects === undefined) return (
+            <div className="w-full min-h-[300px] grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5 gap-y-20 gap-x-[10px] justify-items-center" data-cy="project-list-skeleton">
+                {[...Array(20)].map((_, i) => (
+                    <div key={i} className="flex flex-col rounded-lg overflow-hidden animate-pulse w-full min-w-[160px] max-w-[95vw] sm:min-w-[180px] sm:max-w-xs p-2">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)